script.js

```diff
@@ -1,12 +1,64 @@
+/* Homepage Functions */
+function on() {
+  document.getElementById("overlay").style.display = "block";
+}
 
+function off() {
+  document.getElementById("overlay").style.display = "none";
+}
 
-function on() {
-    document.getElementById("overlay").style.display = "block";
-  }
-  
-  function off() {
-    document.getElementById("overlay").style.display = "none";
-  }
+/* Dynamic Project Functions */
+function projectOn() {
+  document.getElementById("output").style.display = "block";
+}
+
+function projectOff() {
+  document.getElementById("output").style.display = "none";
+}
 
+
+
+function getProject(a) {
+  projectOn();
+fetch("projects.json")
+.then((res) => res.json())
+.then((data) => {
+ output = `
+ <div class="project_overlay">
+  <div class="project_structure_overlay">
+      <div> 
+          <button onclick="projectOff()" class="project_overlay_btn_close"> X </button>
+      </div>
+      <div class="project_heading_overlay">
+              <h2 class="title_overlay"> ${data[a].name} </h2>
+              <ul class="project_body_title_list">
+                  <li class="title_list_main_overlay">${data[a].projectClient}</li>
+                  <li><img class="counter_overlay" src="./Assets/Counter.png" >${data[a].projectInfo[0]}</li>
+                  <li><img class="counter_overlay" src="./Assets/Counter.png" >${data[a].projectInfo[1]}</li>
+              </ul>
+      </div>
+      <div> <img class="image_ovlerlay" src=${data[a].image} alt="snapshoot"> </div>
```

Comment:
- [ ] Great job working on the design but I have small noted I see it would be great if you fix it here the image doesn't have the same width as the figma template so kindly make it to be the same width 🔥🚀😊

- [ ] For the (x) button I recommend tto move it to the end so look like th figma design 😊👍🏻♻

- Your design
![image](https://user-images.githubusercontent.com/58553711/202314558-2520da3e-51b7-4b41-b9f3-885e5928d2ff.png)


- Figma 
![image](https://user-images.githubusercontent.com/58553711/202314468-23503b95-e0f2-48fc-89f1-9c3ad4684122.png)